openedx/openedx-platform
# feat: Remove block_structure.invalidate_cache_on_publish waffle flag …

Description:

As Per this DEPR: https://github.com/openedx/public-engineering/issues/33
Removing: Toggle labeled 'block_structure.invalidate_cache_on_publish'. The toggle has the following annotation a :

When enabled, the block structure cache is invalidated when changes to courses are published. If block_structure.storage_backing_for_cache is active, all block structures related to the published course are also cleared from storage.

Supporting information:
This was a temporary waffle and is no longer needed as the original jeera issue https://openedx.atlassian.net/browse/DEPR-144

Testing instructions

as per the ticket, this waffle flag needs to be DEPR and not used longer.
we install tutor and then remove this code into edx-platfrom first in our local system.
Create a new course in the studio and add videos, textbooks and then publish it.
after publishing we verify that they get updated properly in the LMS courseware or not.
this is an update in LMS and we also add more than three courses check this properly and that work fine.
